### PR TITLE
feat(variant-formatter): add variant formatter

### PR DESCRIPTION
**Pull request by fdlk**
Testing on variants.edge.molgenis.org

#### Checklist
- [x] Functionality works & meets specifications (tested on rancher)
- [x] Templates reviewed
- [x] Added values to questions
- [x] Bumped the chart version
- [x] Updated appVersion (if needed)

### CODE REVIEW

File: charts/variant-formatter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Format variants using biocommons tools
+name: variant-formatter
+home: https://github.com/biocommons/hgvs
+version: 0.0.10
```
Comment (sidohaakma) [inline charts/variant-formatter/Chart.yaml]: Maybe bump to 0.1.0 for production usage?

File: charts/variant-formatter/questions.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    label: "Enable ingress"
+    type: boolean
+    default: true
+    description: "Make accessible from outside the cluster?"
+    required: true
+  - variable: "ingress.hosts[0].host"
+    show_if: "ingress.enabled=true"
+    label: hostname
+    type: hostname
+    default: "variants.dev.molgenis.org"
```
Comment (sidohaakma) [inline charts/variant-formatter/questions.yml]: Always to prod: variants.molgenis.org

File: charts/variant-formatter/values.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    storageClass: ""
+    accessMode: ReadWriteOnce
+    size: 10Gi
+    # existingClaim: ""
+
+replicaCount: 1
+
+image:
+  # TODO: Update after merging https://github.com/molgenis/molgenis-ops-docker/pull/34
+  repository: fdlk/variant-formatter
```
Comment (sidohaakma) [inline charts/variant-formatter/values.yaml]: update to production tag

File: charts/variant-formatter/Chart.yaml (hunk 1 above)
Comment (fdlk) [reply charts/variant-formatter/Chart.yaml]: done!

**Pull request merged**